src/cc1101.cpp

```diff
@@ -577,9 +578,14 @@ void cc1101::getRxFifo(uint16_t Boffs) {           // xFSK
 		if (fifoBytes > 0) {
 			uint8_t marcstate;
 			uint8_t RSSI = cc1101::getRSSI();
-      		
-			// My own special debug setting
 			int8_t freqErr = cc1101::readReg(0x32, CC1101_READ_BURST); // 0x32 (0xF2): FREQEST – Frequency Offset Estimate from Demodulator
+			if (AfcEnabled == 1) {
```

Comment:
Ich war hier unsicher, ob wir das wirklich in die Firmware einbauen oder nicht den Wert von FREQEST an FHEM übergeben, damit wir über FHEM das Register bei Bedarf mittels SET xxx setzen könnten. 

Die Variable hatte ich freqest benannt, wieso heisst die jetzt freqErr?

Reply:
> Ich war hier unsicher, ob wir das wirklich in die Firmware einbauen oder nicht den Wert von FREQEST an FHEM übergeben, damit wir über FHEM das Register bei Bedarf mittels SET xxx setzen könnten.

Da sich der Wert von FREQEST immer mal ändert, ist FHEM dann dauernd damit beschäftigt, irgendwelche Registerwerte zum SIGNALduino zu senden. Das geht auch ziemlich langsam und damit dürfte keine optimale Regelung erfolgen.

> Die Variable hatte ich freqest benannt, wieso heisst die jetzt freqErr?

Wir hatten sie halt so benannt, können wir auch wieder ändern.

Reply:
Ja, das könnte ein Thema sein.